packages/openpmd-api/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: openpmd-api
+  version: 0.14.4
+
+source:
+  url: https://github.com/openPMD/openPMD-api/archive/0.14.4.tar.gz
+  sha256: 42b7bcd043e772d63f0fe0e5e70da411f001db10096d5b8be797ffc88e786379
+
+requirements:
+  run:
+    - python
+    - numpy
+    - libhdf5
+
+test:
+  imports:
+    - openpmd_api
+
+build:
+  library: true
```

Comment:
Help wanted: Not sure what that means :)

This is a C++ library with Python bindings as a module. The primary need here is to use the Python module.

Reply:
See https://pyodide.org/en/latest/development/meta-yaml.html#build-library

If the package doesn't have a  `setup.py`  you would want to set it to True.

Reply:
Thanks, ok!
I do have a `setup.py` for pip-based installing, but it does not export all the extra CMake flags that we need for the webassembly cross-compile here :)

Reply:
Well to be clear, you are only supposed to set this to be true if your package is a static library intended to be linked by some particular other package. It isn't appropriate for a Python package.

Reply:
```suggestion
```